mlir/test/Conversion/MathToROCDL/math-to-rocdl.mlir

```diff
@@ -513,3 +513,44 @@ module {
     "test.possible_terminator"() : () -> ()
   }) : () -> ()
 }
+
+// -----
+
+module @test_module {
+  // CHECK-LABEL: func @math_sin_vector_1d
+  func.func @math_sin_vector_1d(%arg : vector<4xf16>) -> vector<4xf16> {
+    // CHECK: llvm.extractelement {{.*}} : vector<4xf16>
```

Comment:
Do we want to check to see these actually getting passed in to a `sin` call?

Reply:
Done!